game_board.py

```diff
@@ -0,0 +1,199 @@
+import curses
+import os
+import time
+
+# The path of the textfile of the map
+map_path = 'Ascii board.txt'
+
+def read_map(path):
+    try:        
+        board = open( path,'r')
+        board.close 
+
+        board_txt = board.read()
+        return board_txt
+    except IOError:
+        print("Can not find the file" )
+
+
+
+
+def map_cord(str_board):
+    """
+    Reads the string from left to right and appends every char to list with its x any y pos
+    the top of the menu. 
+    Returns a list of lists where every list has [char,xPos,Ypos]
+    Example "X  " => [['X', 0, 0], [' ', 1, 0], [' ', 2, 0]...,]
+
+    Keyword arguments:
+    str_board -- the string of the map
```

Comment:
Should it say the following?
```suggestion
    str_board -- the string version of the map
```